### PR TITLE
[1252] add course delete and withdraw pages

### PR DESCRIPTION
**Pull request by dankmitchell**
### Context
Withdrawing and delete courses

### Changes proposed in this pull request
- Add /withdraw
- Add /delete

### Guidance to review
Find a course on the frontend app and navigate to `/courses/XXX/withdraw` and `/courses/XXX/delete`

# After
localhost_3000_organisations_2AT_courses_3CXL_withdraw (1)
localhost_3000_organisations_2AT_courses_3CXL_delete

### CODE REVIEW

**Comment by dankmitchell** [comment]
Once this is merged https://github.com/DFE-Digital/manage-courses-frontend/pull/150 I can do some cleanup and hopfully the `courses#index` spec will fix itself because the current implementation is flaky...

File: app/views/courses/delete.html.erb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      <p class="govuk-body">Deleting a course is permanent – you can’t undo it. If you need to recreate a course after deleting it, it will have a new course code.</p>
+
+      <h2 class="govuk-heading-m">Contact support to delete this course</h2>
+
+      <p class="govuk-body">Contact the <%= mail_to 'becomingateacher@digital.education.gov.uk', 'Becoming a teacher team', class: "govuk-link" %> by email to delete this course.</p>
+
+      <p class="govuk-body">In your email include:</p>
+      <ul class="govuk-list govuk-list--bullet">
+        <li>course code</li>
+        <li>couese title</li>
```
Comment (defong) [inline app/views/courses/delete.html.erb]: typo!

File: app/views/courses/withdraw.html.erb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      <p class="govuk-body">Once you’ve withdrawn a course, you can’t publish it again this cycle.</p>
+
+      <h2 class="govuk-heading-m">Contact support to withdraw this course</h2>
+
+      <p class="govuk-body">Contact the <%= mail_to 'becomingateacher@digital.education.gov.uk', 'Becoming a teacher team', class: "govuk-link" %> by email to withdraw this course.</p>
+
+      <p class="govuk-body">In your email include:</p>
+      <ul class="govuk-list govuk-list--bullet">
+        <li>course code</li>
+        <li>couese title</li>
```
Comment (defong) [inline app/views/courses/withdraw.html.erb]: typo

File: app/views/courses/delete.html.erb [hunk 3]
```diff
@@ -0,0 +1,32 @@
+<%= content_for :page_title, "Delete" %>
```
Comment (fofr) [inline app/views/courses/delete.html.erb]: Can we use the title: "Are you sure you want to delete [name] ([code])?"

File: app/views/courses/delete.html.erb [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+      <ul class="govuk-list govuk-list--bullet">
+        <li>created it by mistake</li>
+        <li>won’t offer it again</li>
+      </ul>
+
+      <p class="govuk-body">Deleting a course is permanent – you can’t undo it. If you need to recreate a course after deleting it, it will have a new course code.</p>
+
+      <h2 class="govuk-heading-m">Contact support to delete this course</h2>
+
+      <p class="govuk-body">Contact the <%= mail_to 'becomingateacher@digital.education.gov.uk', 'Becoming a teacher team', class: "govuk-link" %> by email to delete this course.</p>
```
Comment (fofr) [inline app/views/courses/delete.html.erb]: Can we put the course code and name into the email subject automatically?

File: app/views/courses/withdraw.html.erb [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+      <ul class="govuk-list govuk-list--bullet">
+        <li>immediately remove it from Find</li>
+        <li>close applications for it (this happens when UCAS Apply updates overnight)</li>
+      </ul>
+
+      <p class="govuk-body">Once you’ve withdrawn a course, you can’t publish it again this cycle.</p>
+
+      <h2 class="govuk-heading-m">Contact support to withdraw this course</h2>
+
+      <p class="govuk-body">Contact the <%= mail_to 'becomingateacher@digital.education.gov.uk', 'Becoming a teacher team', class: "govuk-link" %> by email to withdraw this course.</p>
```
Comment (fofr) [inline app/views/courses/withdraw.html.erb]: Can we put the course code and name into the email subject automatically?

File: app/views/courses/withdraw.html.erb [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+      <p class="govuk-body">In your email include:</p>
+      <ul class="govuk-list govuk-list--bullet">
+        <li>course code</li>
+        <li>course title</li>
+        <li>a reason why this course needs to  be withdrawn</li>
```
Comment (fofr) [inline app/views/courses/withdraw.html.erb]: (double space)

File: app/views/courses/delete.html.erb (hunk 4 above)
Comment (defong) [reply app/views/courses/delete.html.erb]: same with withdraw i guess

**Pull request merged**